pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -676,6 +678,8 @@ async def run_repomdrecord_download(name, location_href, downloader):
                             raise HTTPNotFound(
                                 reason=_("File not found: {}".format(exc.request_info.url))
                             )
+                        except FileNotFoundError:
```

Comment:
Who ends up seeing this? Are they expecting HTTPNotFound? Will they take the same path on a FileNotFoundError ?

Reply:
Nobody catches it, it's fatal to the sync.

Reply:
Gotcha, makes sense.